snaps/index.mdx

```diff
@@ -81,20 +81,20 @@ Features include:
     },
     {
       href: "reference/exports#new-in-metamask-flask-transaction-severity-level",
-      title: "Transaction Severity",
+      title: "Transaction severity",
       description: "Add extra friction to the transaction flow if a transaction looks risky.",
       flag: "Flask only"
     },
     {
       href: "reference/exports#onnamelookup",
-      title: "Name Resolution",
-      description: "Resolve names to addresses and vice-versa.",
+      title: "Name resolution",
+      description: "Resolve names to addresses and vice versa.",
       flag: "Flask only"
     },
     {
       href: "reference/rpc-api#snap_getlocale",
       title: "Localization",
-      description: "Translate your Snap UI based on the user's locale.",
+      description: "Translate your snap UI based on the user's locale.",
```

Comment:
Hi @alexandratran, we have a ton of cleanup to do on this but to be clear a Snap is always capitalized.

Reply:
Okay, can we raise as a separate issue/PR to change this across the docs? Right now it is always lowercase when referencing a single snap.

Reply:
Yes that would be the way to solve it